test/replication-luatest/gh_5295_split_brain_test.lua

```diff
@@ -161,6 +161,10 @@ g.test_bad_confirm_old_term = function(cg)
     reconnect_and_check_split_brain(cg)
 end
 
+g.before_test('test_good_confirm_old_term', function(cg)
+    cg.main:update_box_cfg{replication_synchro_timeout = 1000}
+end)
+
 -- Obsolete sync transaction confirmation might be fine when it doesn't
 -- contradict local history.
 g.test_good_confirm_old_term = function(cg)
```

Comment:
Wouldn't it be easier to just move this single line to the beginning of the test's body? Same for the 'after test' callback. I mean, since the appearance of luatest in our life I am wondering what is the sense of these callbacks, if it is always easier to just write them directly in the test body. No exceptions. What is your opinion?

Reply:
I personally use the callbacks to clarify that this is setup code not related to the test logic so that a reader can focus on the test logic rather than the test setup.

Reply:
The setup is related to the test logic. If the timeout is small, the test wouldn't work, would it? Which makes it a part of the test which is necessary to know. Anyway, thanks for the explanation.